### DISCUSSION TITLE
Pad 上出现红字“For Evaluation Purposes Only”

### DISCUSSION THREAD

**Discussion by Lamagentle** [zotero-chinese/wiki#325] [求助 / Q&A]
在小米pad上的pdf，同步下载下来后，上面有一行红字“For Evaluation Purposes Only， remove with TRIAL key”有没有大神知道怎么办
![8b66ebb5f642e99c290386888773484](https://github.com/user-attachments/assets/5547d126-e105-4ebe-a64e-515a01f52a4b)

**Comment by northword** [comment]
你使用的是测试版本的，你要从google play去下载正式版本就会没有，但谷歌play中这个还在内测。
